generators/app/index.js

```diff
@@ -49,6 +49,7 @@ module.exports = Generator.extend({
 
   default: function () {
     var cmd = [
+      'rm -f master.tar.gz pattern-lab-starter-master patternlab',
```

Comment:
None of our "generator rerun" functionality checks for uncommitted changes. That seems like a nice next step. The main place I would expect this is from the upcoming generator-outrigger-drupal, which could check to see if the repo is dirty & writeable before executing.